resources/dicts/events/lifegen_events/murder.json

```diff
@@ -362,12 +367,12 @@
         "Like a snake with a charming voice, you guide v_c into the corners of the Clan's borders, giving them a false promise of 'teaching' them a new move only they can witness- what's cooler than training with the leader of the Clan! v_c promises to take it to their grave. Unfortunately, you made sure they followed your exact orders.",
-        "You offer to take v_c onto a grand and exciting adventure out of the nursery. They agree, exhilarated at the thought of accompanying the leader on a journey. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind as their startled cries call out your name from behind. You loop back to camp, alone, and no one asks where you've been.",
+        "You offer to take v_c onto a grand and exciting adventure out of the nursery. They agree, exhilarated at the thought of accompanying the leader on a journey. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been.",
         "You tell v_c to follow you, saying that you want to show them a shiny treasure you found recently. They happily agree and the two of you sneak out of camp. You lead v_c to a foxes' den, and they ask you what the smell is with a wrinkled nose. You tell them not to worry about it and push them forward, saying that the treasure is inside. With an anxious look at you, they slowly head in. You don't stay to listen to the foxes' feasting.",
         "You ask v_c if they want to go on a secret mission with you, and they jump excitedly at the opportunity to help the leader. You lead them out of camp and head towards the Thunderpath together. When you start to hear a monster in the distance, you pressure v_c into walking out on the path, saying that you'll be right behind them. As the monster zooms toward the two of you, you quickly run back to the other side of the Thunderpath while v_c is frozen in the headlights. You don't bother looking back.",
-        "You bring v_c a piece of crow-food that you found by the side of the Thunderpath, claiming that it's from a special type of squirrel. They wrinkle their nose at it, but you encourage them to eat it, saying that it's about time they had proper food instead of milk. Desperate to gain your approval, they swallow it down with difficulty, and later you hear v_c's pained wails from the medicine den. Their body is cold and frothing at the mouth when you bury them."
+        "You bring v_c a piece of crow-food that you found by the side of the Thunderpath, claiming that it's from a special type of squirrel. They wrinkle their nose at it, but you encourage them to eat it, saying that it's about time they had proper food instead of milk. Desperate to gain your approval, they swallow it down with difficulty, and after, never wake up."
```

Comment:
this can be changed to match the previous one

"You bring v_c a piece of crow-food that you found by the side of the Thunderpath, claiming that it's from a special type of squirrel. They wrinkle their nose at it, but you encourage them to eat it. Desperate for your approval, they don't wake up ever again."